### DISCUSSION TITLE
Fit sine-wave with a staircase function

### DISCUSSION THREAD

**Discussion by charlesbmi** [staircase-dev/staircase#180] [Q&A]
This library looks great. I came across it looking for a library to fit staircase functions to approximate continuous waveforms. Is there a good library or paper for that?

**Comment by venaturum** [comment]
Hi @charlesbmi, you could try [hotstepper](https://github.com/TangleSpace/hotstepper) which was written by an ex-colleague of mine.  It doesn't look to be actively maintained but it may do what you're after.

**Reply by charlesbmi** [reply]
Thanks! This looks great, appreciate the pointer
